app/src/main/res/layout/fragment_saving_account_details.xml

```diff
@@ -2,8 +2,9 @@
 <android.support.v4.widget.NestedScrollView
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:layout_height="wrap_content"
-    android:layout_width="match_parent">
+    android:layout_height="match_parent"
```

Comment:
@sashank27 Why you made the wrap_content to match_parent?

Reply:
@luckyman20 making it match_parent and doing `android:fillViewport="true"` enables the online layout to implement correct scrolling behaviour, and also makes the offline layout centred.